server/utils/nunjucksSetup.ts

```diff
@@ -92,6 +100,10 @@ export default function nunjucksSetup(app: express.Express, applicationInfo: App
     }
   })
 
+  njkEnv.addFilter('prisonerNameFirstLast', prisoner => {
```

Comment:
I've made this filter specific to **prisoner** names as visitor names don't need to be processed (they don't come through all upper-case).